Library/Homebrew/test/gpg_spec.rb

```diff
@@ -12,8 +12,15 @@
 
         shutup do
           subject.create_test_key(dir)
+          gpg = subject::GPG_EXECUTABLE
+          @version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
```

Comment:
Maybe adding a `version` method to the `Gpg` class would make this clearer and more robust than a `start_with?` check. Otherwise this seems fine, I don't see anything that could even be stubbed here, since it comes down to the version of the installed `gpg` executable.